### PR TITLE
CSI 4731 servicer modify vg implementation

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: controllers/servers/csi/csi_controller_server.py [hunk 1]
```diff
@@ -620,6 +620,57 @@ def DeleteVolumeGroup(self, request, _):
...
+
+            volume_group = self._get_volume_group(array_mediator, volume_group_id)
+
+            volumes_in_volume_group = volume_group.volumes
+            volume_ids_in_volume_group = [volume.id for volume in volumes_in_volume_group]
+            volume_ids_in_request = []
+            for volume_id in request.volume_ids:
+                volume_id_info = utils.get_volume_id_info(volume_id)
+                volume_ids_in_request.append(volume_id_info.ids.uid)
+
```
Comment (ELENAGER) [inline controllers/servers/csi/csi_controller_server.py:667]: Is it possible to add functions like 
volume_ids_in_request =build_volume_ids_in_request(request) 
and 
volume_ids_in_volume_group = build_volume_ids_in_group(volume_group)?

Comment (ArbelNathan) [reply controllers/servers/csi/csi_controller_server.py:667]: for what reason? how is it different?

Comment (ELENAGER) [reply controllers/servers/csi/csi_controller_server.py:667]: I think it will be cleaner to move it into separate function. Why not?

Comment (matancarmeli7) [reply controllers/servers/csi/csi_controller_server.py:667]: I agree with @ELENAGER

Comment (ArbelNathan) [reply controllers/servers/csi/csi_controller_server.py:667]: I really don't understand, how is it gonna look?

Comment (ELENAGER) [reply controllers/servers/csi/csi_controller_server.py:667]: Something like this:
volume_ids_in_request = self._build_volume_ids_in_request(request)

Comment (ArbelNathan) [reply controllers/servers/csi/csi_controller_server.py:667]: for where?
what in _build_volume_ids_in_request?
what line is it even for?
what is it building?

Comment (matancarmeli7) [reply controllers/servers/csi/csi_controller_server.py:667]: I think we can create two functions:
```
def _get_volume_ids_in_request(self, request):
volume_ids_in_request = []
for volume_id in request.volume_ids:
    volume_id_info = utils.get_volume_id_info(volume_id)
    volume_ids_in_request.append(volume_id_info.ids.uid)
return volume_ids_in_request
```

```
def _get_volume_ids_in_volume_group(self, volume_group):
volumes_in_volume_group = volume_group.
return [volume.id for volume in volumes_in_volume_group]

```

Comment (matancarmeli7) [reply controllers/servers/csi/csi_controller_server.py:667]: and in the `ModifyVolumeGroup` function, we will have two lines that each of them calls each function

Comment (ArbelNathan) [reply controllers/servers/csi/csi_controller_server.py:673]: tnx, tnx

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**